llpc/lower/llpcSpirvLowerGlobal.cpp

```diff
@@ -1931,8 +1947,20 @@ void SpirvLowerGlobal::lowerBufferBlock() {
             continue;
 
           usesToReplace.push_back(inst);
+          insertPoint = findNearestCommonDominator(insertPoint, inst, domTree);
```

Comment:
I am not sure if the `inst` might sit in the newly created return block during `lowerOutput()`? If this is possible, I think we may need to update the dominator tree there.

Reply:
I don't know if this can happen. I haven't seen any CTS failures caused by it. (The CI failures are "Flake" not "Fail" and anyway I can't reproduce them on my own Navi21 machine.)